model-optimizer/extensions/ops/roifeatureextractor_onnx.py

```diff
@@ -26,23 +26,19 @@ def __init__(self, graph, attrs):
         mandatory_props = dict(
             type=__class__.op,
             op=__class__.op,
-            version='experimental',
+            version='opset6',
             infer=__class__.infer,
-            in_ports_count=5,
             out_ports_count=2,
         )
 
         super().__init__(graph, mandatory_props, attrs)
 
     def backend_attrs(self):
         return [
-            'distribute_rois_between_levels',
```

Comment:
Please, check what will happen if you read the model with experimental `ExperimentalDetectronROIFeatureExtractor` using the MO IR reader and then save it. What will be the value of `version` attribute for this operation in a new IR?

Reply:
If we change lines 334-335 of model-optimizer/mo/utils/ir_engine/ir_engine.py as in PR https://github.com/openvinotoolkit/openvino/pull/3814, tests with MO IR Reader is passed, and `version` in IR restored by MO IR Reader is equal to `version` from old IR.